docs/aop/user/apps/usercentrics.md

```diff
@@ -0,0 +1,138 @@
+7. In the top right corner of the Usercentrics app details page, click **Configure**.
+8. In the *Configure* pane, in *Global Settings* by default, *[Enable Smart Data Protector]*(#smart-data-protector) is selected. You can either leave this setting or select *[Enable Direct Integration (Works only with Google Tag manager)]*(#google-tag-manager).
+9. Select the store and insert the setting ID from step 5.
+10. To activate Usercentrics for the selected store, select *Is active*.
+11. Optional: To add more stores with the same or different setting IDs, click *Add store configuration*.
+
+
+That's it. You have integrated the Usercentrics app into your store. The app changes to *Connected* on the apps catalog page. Now, the cookie consent popups should be displayed on every page of the Storefront.
+
+### Global settings configuration
+
+There are three ways to integrate Usecentrics: via direct integration, via the Usercentrics Smart Data Protector, and integration with Google Tag Manager. In the Spryker Back Office, you can select either Smart Data Protector or Google Tag Manager. At the same time, the Smart Data Protector is the preferred and default setting.
+
+{% info_block infoBox "Info" %}
+
+All of the three approaches require you to get the setting ID for your store in the [Usercentrics Admin Interface](https://admin.usercentrics.eu/) and configure the data processing services on page **Service Settings -> Data Processing Services**.
+
+{% endinfo_block %}
+
+#### Direct integration
+
+The direct integration requires mostly manual developer work. You can do the direct integration if you don't use a third-party tracking tool yet.
+
+The direct integration presupposes that you inject the Usercentrics JavaScrip tag into your site with the setting ID.
+	
+Example script tag for the direct integration:
+	
+```
+<script id="usercentrics-cmp" data-settings-id="apXtrZ9ec" src="https://app.usercentrics.eu/browser-ui/latest/bundle.js" defer></script>
+```
+
+where `apXtrZ9ec` is the setting ID that identifies configuration of the data processing services for your store in the Usercentrics Admin Interface.
+
+For the direct integration, you have to programmatically deactivate the JavaScript code of all the tracking tools on the shop pages and give tools the names that match those in the Usercentrics Admin Interface.
+   
+For more details on the Usercetrics direct integration, see the [Usercentrics documentation](https://docs.usercentrics.com/#/direct-implementation-guide).
+
+#### Smart Data Protector
+
+The [Smart Data Protector](https://docs.usercentrics.com/#/smart-data-protector) blocks the data processing services you added to your store and activates them only for customers that gave their consent to do so. The Smart Data Protector is the preferred and default setting to configure Usercentrics. You can apply it if you don't use a third-party tracking tool yet and don't want to use developers' work.
```

Comment:
"don't want to use developers' work" - I know what you mean, but I feel it's not quite there. We should  stress more the "code free" & convenience aspect of it - you don't need a developer!